.github/workflows/ci.yml

```diff
@@ -24,3 +31,16 @@ jobs:
       - run: pnpm install
       - run: pnpm install typescript@${{ matrix.ts }}
       - run: pnpm type-check
+
+  are-the-types-wrong:
+    runs-on: ubuntu-latest
+    needs: run
+    steps:
+      - uses: actions/checkout@v4
+      - run: corepack enable
+      - run: pnpm install
```

Comment:
Are these necessary? They might actually hide errors eg if we accidentally use types from node_modules

Reply:
How would we accidentally use types from `node_modules`?

Reply:
I'm actually not sure how attw checks so maybe it's not possible. 

But since installing here would add a node_modules folder, an import in the package would be able to resolve stuff from node_modules, which we wouldn't want except for prod dependencies - and we don't have any.

Reply:
`attw` inspects the tarball and checks to see if the type definitions are valid based on `package.json`'s `types` and `exports` fields (oversimplifying).

Reply:
Yeah, I think my comment got shifted somehow but what I'm objecting to is checkout/ corepack enable/ pnpm install - even if they doesn't add the risk I suggested they are still unnecessary if we're using pnpx, as far as I can tell.

Reply:
Oh I see, I misunderstood. Okay let me try fixing it.

Reply:
`corepack enable` turned out to be necessary but I got rid of the other stuff.